src/Blocks/sources.jl

```diff
@@ -34,12 +77,19 @@ function Sine(;name,
     amplitude=1,
     phase=0,
     offset=0,
-    start_time=0)
+    start_time=0,
+    smooth=false)
 
     @named output = RealOutput()
     pars = @parameters offset=offset start_time=start_time amplitude=amplitude frequency=frequency phase=phase
+    equation = if smooth == false
+        offset + ifelse(t < start_time, 0, amplitude* sin(2*pi*frequency*(t - start_time) + phase))
+    else
+        δ = 1e-5
```

Comment:
Should this parameter be exposed and documented?

Reply:
I think it should

Reply:
Perhaps yes. Although having δ = 1e-6 or such makes a miniscule change to the result, it can upset high precision tests